README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:

```

Comment:
just looked at `class-transformer-validator` and realize that its not related to routing-controllers and can be used just with validator+class-transformer. But inside it just calls two methods of two libraries in sequence and provide a single function to use, right? If so, then do routing-controllers need it? Probably no, because it does not need such "facade function" because its used only once and is internal. It simply can call two methods of two libraries in sequence by its own. So, now the question is: Does it worth to bind routing-controllers to extra dependency that makes a small amount of work that routing-controllers can do on its own? Answers is in my opinion is no. But this tool can be useful for those who are using it just with validator+class-transfromer, without routing-controllers, as you told.